.travis.yml

```diff
@@ -14,23 +20,13 @@ cache:
         - $HOME/.composer/cache/files
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
     - 7.1
     - hhvm
 
 matrix:
     allow_failures:
         - php: hhvm
         - php: 7.1
-    include:
-        - php: 5.3
-          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
-        - php: 5.6
-          env: DEPENDENCIES=dev
```

Comment:
testing lowest deps and dev deps must be kept (we could change the PHP version to a higher version for dev deps, and the lowest one should stay on the lowest supported PHP version)

Reply:
If it would work I would have kept it.